staging/src/k8s.io/apimachinery/pkg/api/resource/quantity_test.go

```diff
@@ -1207,11 +1207,11 @@ func TestQuantityAsApproximateFloat64(t *testing.T) {
 		{decQuantity(1024, 0, BinarySI), 1024},
 		{decQuantity(8*1024, 0, BinarySI), 8 * 1024},
 		{decQuantity(7*1024*1024, 0, BinarySI), 7 * 1024 * 1024},
-		{decQuantity(7*1024*1024, 1, BinarySI), (7 * 1024 * 1024) * 1024},
-		{decQuantity(7*1024*1024, 4, BinarySI), (7 * 1024 * 1024) * (1024 * 1024 * 1024 * 1024)},
-		{decQuantity(7*1024*1024, 8, BinarySI), (7 * 1024 * 1024) * (1024 * 1024 * 1024 * 1024 * 1024 * 1024 * 1024 * 1024)},
-		{decQuantity(7*1024*1024, -1, BinarySI), (7 * 1024 * 1024) / float64(1024)},
-		{decQuantity(7*1024*1024, -8, BinarySI), (7 * 1024 * 1024) / float64(1024*1024*1024*1024*1024*1024*1024*1024)},
+		{decQuantity(7*1024*1024, 1, BinarySI), (7 * 1024 * 1024) * 10},
+		{decQuantity(7*1024*1024, 4, BinarySI), (7 * 1024 * 1024) * 10000},
+		{decQuantity(7*1024*1024, 8, BinarySI), (7 * 1024 * 1024) * 100000000},
+		{decQuantity(7*1024*1024, -1, BinarySI), (7 * 1024 * 1024) * math.Pow10(-1)}, // '* Pow10' and '/ float(10)' do not round the same way
+		{decQuantity(7*1024*1024, -8, BinarySI), (7 * 1024 * 1024) / float64(100000000)},
```

Comment:
Looks correct, please pull in the changes in https://github.com/kubernetes/kubernetes/pull/104757 (the additional test added to quantity_test.go and the change to scheduler resources_test.go to surface the problem).

The release note needs to be changed to read:

> Scheduler resource metrics over fractional binary quantities (`2.5Gi`, `1.1Ki`) were incorrectly reported using the wrong power.

Reply:
I appreciate your help! 🙇
I pulled in the changes in #104757 and updated the release note.